src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1430,7 +1430,7 @@ internal bool CanSaveAs(object parameters)
         /// <param name="path">The path to the file.</param>
         /// <exception cref="IOException"></exception>
         /// <exception cref="UnauthorizedAccessException"></exception>
-        internal void SaveAs(string path, bool isBackup = false)
+        public void SaveAs(string path, bool isBackup = false)
```

Comment:
Thanks, I have not really thought about this before but seems a good change to me that currently only tests and excel node are using this function, and this API could be open